get_github_coderabbit_review/coderabbit_fetcher/ai/sentiment_analyzer.py

```
@@ -0,0 +1,704 @@
+                self._update_stats(result)
+                return result
+
+            # Default neutral result
+            result = SentimentScore(
+                sentiment_score=0.0,
+                sentiment_label=SentimentLabel.NEUTRAL,
+                confidence=0.1,
+                reasoning="No analysis method available",
+            )
+            self._update_stats(result)
+            return result
```

Ruff (0.13.2)

Consider moving this statement to an `else` block

(TRY300)